### PR TITLE
GBA: add missing entries from no-intro

### PR DESCRIPTION
**Pull request by tjanas**
Resolves https://github.com/SmokeMonsterPacks/EverDrive-Packs-Lists-Database/issues/211

Use latest revision in main directories, move older revisions to Revisions directory.
Fixes some bad dumps (e.g. [Pokemon - Aurora Ticket Distribution (USA)](https://datomatic.no-intro.org/index.php?page=show_record&s=23&n=x131)).

Move Betas to Betas directory.

Adds new `EverDrive GBA/2 Other Regions - A-Z/` directory.

For consistency, missing compilation roms were added to `EverDrive GBA/4 Game Series Collections/Genre/Compilation/` but not to the main directories, as existing compilation roms are currently absent too. At some point, all of them should be added outside of `EverDrive GBA/4 Game Series Collections/`, perhaps as its own subdirectory.

### CODE REVIEW

**Comment by tjanas** [comment]
@frederic-mahe please review when you have some free time.

**Comment by tjanas** [comment]
@frederic-mahe please review when you have some free time.

File: EverDrive Pack SMDBs/EverDrive GBA SMDB.txt [hunk 1]
```diff
@@ -2478,12 +2481,175 @@ d9c8819be3b4ef9a1aa095cc3ac58395273913a394c74b6f7b01c6e3e45d788f	EverDrive GBA/2
...
+de78d62c1d36769f2f02284379fc00a6dafc6980510fdea34fe36dd075635975	EverDrive GBA/2 Other Regions - A-Z/France/Qui Veut Gagner des Millions (France).gba	4e069eb2ded73b07d8b87836a7f2fac05e62db86	3dedf95b5692abf4a3487bb2500dd566	8f33bd3f
+f5d1cbedec7916395f2195fc129dcf1cb46c52563031104ab798fe3125c77ac4	EverDrive GBA/2 Other Regions - A-Z/France/Racing Fever (France).gba	67eb3051c7527692cfd7515ea367796fab3a1362	edfdf49642c0cc0458e0584792f869c5	4f61dedb
+81d3a3d98325192b2f61eba28449a965361353351b7bc3c7199d5637f619fcdc	EverDrive GBA/2 Other Regions - A-Z/France/Razmoket Rencontrent les Delajungle, Les (France).gba	457be286cb043e1a8eaf7a9a7929b2cbfb56fcda	ebb31e3fe75ca8fcae960d3588ef767d	720b7ff2
+95935f35372d244bb87a9d2525e533decd0202e7aed375547296b05594b14dc1	EverDrive GBA/2 Other Regions - A-Z/France/Razmoket, Les - A Moi la Fiesta (France).gba	96a156840eaafb0f7bf78d99dee49c90c0dafd74	365bce291e2ed017ae5585489e036cb2	4130f7d0
+4267fef4a1d3d5325ddc6d1284281def2063c0bb4424a48f118461a30e15bc5a	EverDrive GBA/2 Other Regions - A-Z/France/Razmoket, Les - Voler N'Est Pas Jouer (France).gba	c5a9a630a3f3ebd2ea858d127a1ce830350361ee	5856c031f9a9dc4c3e395fe336858d9d	809a103a
```
Comment (frederic-mahe) [inline EverDrive Pack SMDBs/EverDrive GBA SMDB.txt:2520]: `Razmoket, Les - Voler N'Est Pas Jouer` should be `Razmoket, Les - Voler n'est pas jouer`

File: EverDrive Pack SMDBs/EverDrive GBA SMDB.txt [hunk 2]
```diff
@@ -19,7 +19,7 @@ a75c9d5f1508dd114749c4418900a745fd075e276da09e2f6778069755510f08	EverDrive GBA/1
 eaa3e8311569f86a20aa3e82fc9ab44135a809dc0a177f3961d5d963a83d5d7e	EverDrive GBA/1 USA - A-C/AirForce Delta Storm (USA) (En,Ja,Fr,De).gba	7bc53480a43ada2aad32d798ab00b6e761726728	3f729eba45634801650d0ca02e2ad0bf	ebf757b8
 8faf7635d203fde1b556ebc0c5b9110e03570605b5306f3ec21addddca837337	EverDrive GBA/1 USA - A-C/Aladdin (USA) (En,Fr,De,Es).gba	3cfe653a54f3bc9f3ad20234de680ebfc4145cf6	4bda2a3e0249dfd09330dabc3aa20978	40b383f6
 16117cd914b7088936c827cdeb7dbe5be382148a77add40fdd1c6a63f9667f98	EverDrive GBA/1 USA - A-C/Alex Rider - Stormbreaker (USA).gba	e9ba9d340701173ccd3a4d279630e1541ed96a73	4b5aee04f0f61c5ef9343524bd2ac1d4	cca55dfd
-88465bfb0bde8a9a3df4a6081cc9f8e999fa74c68c1ec91ebc8dfc977be85fd0	EverDrive GBA/1 USA - A-C/Alienators - Evolution Continues (USA, Europe).gba	fb691c5e21fa388d75497e9090db82dec881e422	5eba0ef1c24f1b003b3063beb808ce6c	0d694ca4
+88465bfb0bde8a9a3df4a6081cc9f8e999fa74c68c1ec91ebc8dfc977be85fd0	EverDrive GBA/1 USA - A-C/Alienators - Evolution Continues (USA, Europe) [b].gba	fb691c5e21fa388d75497e9090db82dec881e422	5eba0ef1c24f1b003b3063beb808ce6c	0d694ca4
```
Comment (frederic-mahe) [inline EverDrive Pack SMDBs/EverDrive GBA SMDB.txt:22]: should `[b]` be replaced with `(Rev 1)`?

File: EverDrive Pack SMDBs/EverDrive GBA SMDB.txt (hunk 1 above)
Comment (frederic-mahe) [reply EverDrive Pack SMDBs/EverDrive GBA SMDB.txt:2520]: We use less capitals in French.

**Comment by frederic-mahe** [comment]
Awesome work @tjanas !

File: EverDrive Pack SMDBs/EverDrive GBA SMDB.txt (hunk 2 above)
Comment (tjanas) [reply EverDrive Pack SMDBs/EverDrive GBA SMDB.txt:22]: No, this rom is currently the only known dump and [no-intro has identified it as bad](https://datomatic.no-intro.org/index.php?page=show_record&s=23&n=0175).

File: EverDrive Pack SMDBs/EverDrive GBA SMDB.txt (hunk 1 above)
Comment (tjanas) [reply EverDrive Pack SMDBs/EverDrive GBA SMDB.txt:2520]: Looks like [no-intro](https://datomatic.no-intro.org/index.php?page=show_record&s=23&n=0553) needs a French speaker. :) I'll make the edit.

**Pull request merged**